packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -117,6 +117,28 @@ class RecoveryPhoneHandler {
 
     throw AppError.invalidOrExpiredOtpCode();
   }
+
+  async destroy(request: AuthRequest) {
+    const { uid } = request.auth.credentials as unknown as { uid: string };
+
+    let success = false;
+    try {
+      success = await this.recoveryPhoneService.removePhoneNumber(uid);
+    } catch (error) {
+      throw AppError.backendServiceFailure(
+        'RecoveryPhoneService',
+        'destroy',
+        { uid },
+        error
+      );
+    }
+
+    if (success) {
+      await this.glean.twoStepAuthPhoneRemove.confirmSubmit(request);
```

Comment:
Looks like the backend event for this should be `two_step_auth_phone_remove_success` according to https://docs.google.com/spreadsheets/d/1DgohA2JG8gvFOjk_8zUHOi6DSDSPzqCCwoNVvirDpkA/edit?gid=1073857271#gid=1073857271

Reply:
Thanks